lib/src/test/java/vmath/Vec3Test.java

```diff
@@ -50,7 +50,29 @@ void givenNonZeroVectors_whenMagnitudeCalculated_thenResultsAreCorrect() {
         var v7 = new Vec3(1.0f, 1.0f, 1.0f);
         assertEquals(v7.mag(), 1.7320508075688772f);
     }
+    @Test
+    void givenZeroVector_whenSquaredMagnitudeCalculated_thenResultIsZero() {
+        var v = new Vec3(0.0f, 0.0f, 0.0f);
+        assertEquals(v.magSqr(), 0.0f);
+    }
 
+    @Test
+    void givenNonZeroVectors_whenSquaredMagnitudeCalculated_thenResultsAreCorrect() {
+        var v1 = new Vec3(1.0f, 0.0f, 0.0f);
+        assertEquals(v1.magSqr(), 1.0f);
+        var v2 = new Vec3(0.0f, 1.0f, 0.0f);
+        assertEquals(v2.magSqr(), 1.0f);
+        var v3 = new Vec3(0.0f, 0.0f, 1.0f);
+        assertEquals(v3.magSqr(), 1.0f);
```

Comment:
Please adopt a standard style for this test - in the first two test files there are no empty lines between "assertEquals" and "var v2" (for example), but in the third test file there are empty lines. I would recommend adding these empty lines to all the test files.

Reply:
Dear Valentina, all fixed. Thanks for your review.